packages/contracts/src/framework/plugin/repo/PluginRepo.sol

```diff
@@ -89,14 +85,11 @@ contract PluginRepo is
     /// @param pluginSetup The plugin setup contract address.
     error PluginSetupAlreadyInPreviousRelease(uint8 release, uint16 build, address pluginSetup);
 
-    /// @notice Thrown if the metadata URI is not set for a release.
-    /// @param release the release number for which the metadata URI is not set.
-    /// @param releaseMetadata The release metadata URI.
-    error InvalidReleaseMetadata(uint8 release, bytes releaseMetadata);
+    /// @notice Thrown if the metadata URI is length of 0.
+    error EmptyReleaseMetadata();
 
     /// @notice Thrown if release does not exist.
-    /// @param release The release number of the release that does not exist.
-    error ReleaseDoesNotExist(uint8 release);
```

Comment:
Errors should be verbose, please revert

Reply:
IMO you already know the release that you passed